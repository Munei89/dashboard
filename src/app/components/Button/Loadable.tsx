/**
 * Asynchronously loads the component for HomePage
 */

import { lazyLoad } from 'utils/loadable';

export const StyledButton = lazyLoad(
  () => import('./index'),
  module => module.StyledButton,
);
