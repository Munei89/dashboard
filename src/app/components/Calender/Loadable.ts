/**
 * Asynchronously loads the component for HomePage
 */

import { lazyLoad } from 'utils/loadable';

export const StyledCalender = lazyLoad(
  () => import('./index'),
  module => module.StyledCalender,
);
