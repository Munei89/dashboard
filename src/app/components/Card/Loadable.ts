/**
 * Asynchronously loads the component for HomePage
 */

import { lazyLoad } from 'utils/loadable';

export const StyledCard = lazyLoad(
  () => import('./index'),
  module => module.StyledCard,
);
