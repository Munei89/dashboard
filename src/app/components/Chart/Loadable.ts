/**
 * Asynchronously loads the component for HomePage
 */

import { lazyLoad } from 'utils/loadable';

export const Chart = lazyLoad(
  () => import('./index'),
  module => module.Chart,
);
