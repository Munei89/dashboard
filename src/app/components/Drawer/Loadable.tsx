/**
 * Asynchronously loads the component for HomePage
 */

import { lazyLoad } from 'utils/loadable';

export const StyledDrawer = lazyLoad(
  () => import('./index'),
  module => module.StyledDrawer,
);
