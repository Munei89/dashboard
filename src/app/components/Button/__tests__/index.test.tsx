import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { StyledButton } from '../Loadable';

const renderer = createRenderer();

describe('<StyledButton />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<StyledButton>Button</StyledButton>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
