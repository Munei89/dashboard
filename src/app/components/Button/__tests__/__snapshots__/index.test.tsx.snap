// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledButton /> should render and match the snapshot 1`] = `
<React.Suspense
  fallback={null}
>
  <UNDEFINED>
    Button
  </UNDEFINED>
</React.Suspense>
`;
